### ISSUE TITLE
Colorized output broken in 1.5.3

### ISSUE THREAD

**Issue by swrobel** [code-mancers/invoker#190]
I believe this commit may be the culprit: https://github.com/code-mancers/invoker/commit/1ab0af59365b2bbebee60120bb30543b8a433c23

**Comment by gnufied** [comment]
Yeah  - I disabled colors in logs grabbed in Invoker log file. That was kind of intentional because, ANSI color escape sequences in log file cause unnecessary noise.  We are still printing colored output to terminal though.

**Comment by swrobel** [comment]
I am not seeing color in my terminal output. I reverted back to 1.5.1 and color is working again.

**Comment by gnufied** [comment]
Agreed. my bad!

**Comment by adisos** [comment]
@swrobel @gnufied 
I always use invoker in daemon mode, so any chance I could tail logs in colorized mode?

**Comment by gnufied** [comment]
I can consider it, the reason I removed colorized output from log file is because it seems to confuse tools like grep or if you are reading logs in a tool that is not capable of handling ascii escape sequences.  May be we should revert back colorized output to way it was before and add a flag to disable colors if logs are going to a file?

**Comment by adisos** [comment]
It would be awesome.
